src/components/Attachments/AttachmentView/useAttachmentErrors.ts

```diff
@@ -0,0 +1,44 @@
+import {useCallback, useState} from 'react';
+
+type AttachmentErrors = Record<string, boolean> | Record<string, never>;
```

Comment:
I think you should be able to reuse `AttachmentSource` type here:

```suggestion
type AttachmentErrors = Record<AttachmentSource, boolean>
```

Reply:
<img width="849" alt="NotReally" src="https://github.com/user-attachments/assets/2be29dc7-1e07-4315-871f-97f73aeb33f0" />

Not really, will think of better type tomorrow.

Reply:
nvm, argument for methods is now AttachmentSource